TCAT/Views/BusLocationView.swift

```diff
@@ -98,16 +96,18 @@ class BusLocationView: UIView {
             }
         }
         
+        let newDegrees = Double(degrees) - self.currentBearing
```

Comment:
this is kind of nit-picky but do you need the `self.`?

Reply:
I look putting a little bit of my`self` into my code

Reply:
It was a leftover from being in the `UIView.animate` closure

Reply:
lol okay then yeah `self` is necessary

Reply:
edit: misread your comment but cool 👍